lib/absinthe/type/custom.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Absinthe.Type.Custom do
+  use Absinthe.Schema.Notation
+
+  @moduledoc """
+  This module contains additional data types.
+
+  To use: `import_types Absinthe.Type.Custom`.
+  """
+
+  scalar :datetime, name: "DateTime" do
+    description """
+    The `DateTime` scalar type represents a date and time in the UTC
+    timezone. The DateTime appears in a JSON response as an ISO8601 formatted
+    string, including UTC timezone ("Z").
```

Comment:
It might be helpful to include a brief sentence about the input format's expectations, too. Parsing an input string with a timezone offset other than 0 is considered an error.